src/test/java/homework_3/ex1/Exercise1.java

```diff
@@ -0,0 +1,95 @@
+package homework_3.ex1;
+
+import homework_3.AbstractBaseTest;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+import static homework_3.Constants.*;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+/**
+ * Created by dinar on 23.11.2019.
+ */
+public class Exercise1 extends AbstractBaseTest {
+
+    private StepsEx1 steps;
+    private static final String PATH_TO_PROPERTIES = "src/main/resources/hw3/UserData.properties";
+
+    @BeforeMethod
+    @Override
+    public void setUp() {
+        super.setUp();
+        steps = new StepsEx1(driver);
+    }
+
+    @Test
+    public void testExercise1() throws IOException {
+
+        // 1. open site
+        steps.openEpamJdiSite();
+
+        //2.Assert Browser title ; 5.Assert Browser title
+        assertEquals(driver.getTitle(), "Home Page");
+
+        // 3.Perform login
+        Properties userData = new Properties();
+        userData.load(new FileInputStream(PATH_TO_PROPERTIES));
+        steps.login(userData.getProperty("login"), userData.getProperty("password"));
+
+        // 4.Logined user's name is display correctly
+        steps.usernameOnTheHomePageShouldBe("PITER CHAILOVSKII");
+
+        //6.Menu buttons are displayed and have proper texts "HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS"
+        assertTrue(steps.getItemOfHeaderMenu(HOME).isDisplayed());
+        assertTrue(steps.getItemOfHeaderMenu(CONTACT_FORM).isDisplayed());
+        assertTrue(steps.getItemOfHeaderMenu(SERVICE).isDisplayed());
+        assertTrue(steps.getItemOfHeaderMenu(METALS_AND_COLORS).isDisplayed());
+
+        assertEquals(steps.getItemOfHeaderMenu(HOME).getText(), "HOME");
```

Comment:
Why do you decide use mix approach? I could see assertion step in the `StepEx1` and I could see assertions here

Reply:
Fix(move all assertion to step class)